cmd/talosctl/pkg/talos/action/tracker.go

```diff
@@ -189,6 +191,38 @@ func (a *Tracker) Run() error {
 		Backoff:           backoff.Config{},
 		MinConnectTimeout: 20 * time.Second,
 	}))
+
+	err = multierror.Append(err, eg.Wait())
+
+	if !a.debug {
+		return err
+	}
+
+	var failedNodes []string
+
+	failedNodesToDmesgs.Range(func(key, value any) bool {
+		failedNodes = append(failedNodes, key.(string))
+
+		return true
+	})
+
+	if len(failedNodes) > 0 {
+		sort.Strings(failedNodes)
+
+		fmt.Printf("console logs for nodes %v:\n", failedNodes)
```

Comment:
```suggestion
		fmt.Printf("console logs for nodes %q:\n", failedNodes)
```

Having " is nice here.

Reply:
done